taker-frontend/src/authentication/useAuth.tsx

```diff
@@ -47,6 +47,7 @@ export function AuthProvider({
             .then(({ authenticated, first_login }: Authenticated) => {
                 setAuthenticated(authenticated);
                 setFirstLogin(first_login);
+                localStorage.setItem("oldUser", "true");
```

Comment:
This won't work, because we always get here even if the user never logged in.

Reply:
Why didn't you go for a simple tooltip?

Reply:
because :
1) making the user type the default password manually is an unnecessary hurdle, it's "making the user think"
2) you don't want to display the tooltip / footer whenever the user is using the non-default password; because then it is just uselessly spamming the user